ocrd-fileformat-transform

```diff
@@ -95,6 +95,11 @@ main () {
             continue
         fi
 
+        if [ -e "$out_file" ];then
```

Comment:
I think that's the other way around? This tests for existence, did you mean to negate that?

Reply:
oh, yes of course. 😊

Reply:
55f6b37